### ISSUE TITLE
Anyone to exclude methods included via external traits?

### ISSUE THREAD

**Issue by ahungry** [johnbillion/php-docs-standards#4]
For some things, like including the Symfony PSR logger, the setLogger method there doesn't have proper docblock syntax, and this is detected by the test as a failure.

However, short of overidding the method in the class making use of said trait (and other than updating the source of the issue) this will cause an error to be produced as a result of something the maintainer of the class has little control over.

Is there (or can there be) an optional setting to exclude certain methods from being analyzed?

**Comment by ahungry** [comment]
It seems like it may be that any methods inherited from traits are considered to have a blank docstring actually...

**Comment by johnbillion** [comment]
Yeah this is definitely something I want to fix. The lib should only test methods of the specified class and not any of its inherited methods.
